proparse/src/main/java/org/prorefactor/treeparser01/TP01Support.java

```diff
@@ -1150,23 +1150,23 @@ private Block pushBlock(Block block) {
     return block;
   }
 
-  public SymbolScope getCurrentScope() {
+  public TreeParserSymbolScope getCurrentScope() {
     return currentScope;
   }
 
-  public SymbolScopeRoot getRootScope() {
+  public TreeParserRootSymbolScope getRootScope() {
     return rootScope;
   }
 
   private void classStateInherits(JPNode classNode, JPNode inheritsTypeNode) {
     LOG.trace("Entering classStateInherits {} {}", classNode, inheritsTypeNode);
     String className = inheritsTypeNode.attrGetS(IConstants.QUALIFIED_CLASS_STRING);
-    SymbolScopeSuper cachedCopy = null;
+    TreeParserSuperSymbolScope cachedCopy = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "cachedCopy" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1481)